beacon-chain/core/helpers/committee.go

```diff
@@ -22,6 +22,11 @@ import (
 var committeeCache = cache.NewCommitteesCache()
 var proposerIndicesCache = cache.NewProposerIndicesCache()
 
+// ResetCommitteeCache is used from other packages (in tests) to reset the cache.
+func ResetCommitteeCache() {
```

Comment:
why do we need this ?

Reply:
Because, when we run multiple tests that rely on state validators, we can have only a single state (all further checks are pulling validators from the global cache). See https://github.com/prysmaticlabs/prysm/blob/80ab9201b35b543c245d190d0216eac9863155a1/beacon-chain/core/helpers/weak_subjectivity_test.go#L64

Reply:
Cant we use this method then ?
```
ClearCache()
```

Reply:
Sounds reasonable. Will update the code.